common/lib/common-utils/src/delay.ts

```diff
@@ -0,0 +1,10 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Returns a promise that resolves after timeMs
```

Comment:
consider adding the following to comment:
@param timeMs - time in milliseconds to wait